### PR TITLE
Add playtest announcement

### PR DESCRIPTION
**Pull request by PunkPun**
Ready for reviews

### CODE REVIEW

File: gamenews [hunk 1]
```diff
@@ -1,9 +1,4 @@
...
-	Author: Kaution
-	Content: The first tournament on the new release is upon us: on March 18 at 17:00 UTC we will have our 6th edition of the Kautious Kup for Red Alert. If you would like to sign up or find more information, head over to the official Discord server and check out \#competitions-events.
-	
-2023-02-25 18-00-00:
-	Title: Release 20230225
+2023-07-31 07-00-00:
+	Title: Playtest 20230731
 	Author: PunkPun
-	Content: Today we have a special occasion on our hands. We have been waiting for this moment for almost two years: A new stable OpenRA Release is out now, containing several major new features and many small-scale polish improvements. Head to www.openra.net for the full news post and to download the new release.
+	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • And many more small fixes and features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on forum, Discord, or GitHub.Happy Playtesting!
```
Comment (Mailaender) [inline gamenews:4]:
```suggestion
	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • And many more small fixes and features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on forum, Discord, or GitHub. Happy Playtesting!
```
?

File: gamenews [hunk 2]
```diff
@@ -1,9 +1,4 @@
...
-	Author: Kaution
-	Content: The first tournament on the new release is upon us: on March 18 at 17:00 UTC we will have our 6th edition of the Kautious Kup for Red Alert. If you would like to sign up or find more information, head over to the official Discord server and check out \#competitions-events.
-	
-2023-02-25 18-00-00:
-	Title: Release 20230225
+2023-07-31 07-00-00:
+	Title: Playtest 20230731
 	Author: PunkPun
-	Content: Today we have a special occasion on our hands. We have been waiting for this moment for almost two years: A new stable OpenRA Release is out now, containing several major new features and many small-scale polish improvements. Head to www.openra.net for the full news post and to download the new release.
+	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • And many more small fixes and features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on forum, Discord, or GitHub. Happy Playtesting!
```
Comment (abcdefg30) [inline gamenews:4]: `Notable changes include: ... And many more small fixes and features` is weird. Don't we have anything more specific ("notable")?

Comment (PunkPun) [reply gamenews:4]: not really, this release is filled with small fixes and small features. And a start or continuation of large refactors

File: gamenews [hunk 3]
```diff
@@ -1,9 +1,4 @@
...
-	Content: The first tournament on the new release is upon us: on March 18 at 17:00 UTC we will have our 6th edition of the Kautious Kup for Red Alert. If you would like to sign up or find more information, head over to the official Discord server and check out \#competitions-events.
-	
-2023-02-25 18-00-00:
-	Title: Release 20230225
+2023-07-31 07-00-00:
+	Title: Playtest 20230731
 	Author: PunkPun
-	Content: Today we have a special occasion on our hands. We have been waiting for this moment for almost two years: A new stable OpenRA Release is out now, containing several major new features and many small-scale polish improvements. Head to www.openra.net for the full news post and to download the new release.
+	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • Red Alert balance changes\n • And many more small fixes and features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on forum, Discord, or GitHub.Happy Playtesting!
```
Comment (abcdefg30) [inline gamenews:4]:
```suggestion
	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • Red Alert balance changes\n • And many more small fixes and features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on forum, Discord, or GitHub. Happy Playtesting!
```
Also the issue with the list is still present. I suggest moving `And many more small fixes and features` outside of it.

Comment (PunkPun) [reply gamenews:4]: oh mb, regressed it in the last push

Comment (PunkPun) [reply gamenews]: What list?

Comment (abcdefg30) [reply gamenews]: The `Notable changes include` list.

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+disqus_id: "playtest-20230731"
+permalink: "/news/playtest-20230731/"
+---
+
+As summer is in full swing, we are excited to kick off the playtest series for the upcoming OpenRA release! This time around, it is all about spring cleaning: squashing bugs, implementing minor requested features, and preparing for larger reworks. But fear not! We still have a bunch of cool stuff lined up just for you! Without further ado, here are some notable changes you can expect in [Playtest 20230731]({{ '/download' | relative_url }}):
+
+* Added new Covert Ops missions for Tiberian Dawn: "Eviction Notice" and "Twist of Fate"
+* Added new Red Alert missions "Allies10b" and "Soviet13a" and the Counter-Strike mission "Mousetrap"
+* Fixed several palette remapping issues and restored player color brightness remapping
+* Implemented a new custom Lua wrapper
```
Comment (pchote) [inline news/_posts/2023-07-31-playtest-20230731.md:15]: I'm fairly sure that most people reading this won't understand what this means. I suggest rewording to something like "Improved error reporting for map scripters".

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+created_at: 2023-07-31 07:00 +0000
+disqus_id: "playtest-20230731"
+permalink: "/news/playtest-20230731/"
+---
+
+As summer is in full swing, we are excited to kick off the playtest series for the upcoming OpenRA release! This time around, it is all about spring cleaning: squashing bugs, implementing minor requested features, and preparing for larger reworks. But fear not! We still have a bunch of cool stuff lined up just for you! Without further ado, here are some notable changes you can expect in [Playtest 20230731]({{ '/download' | relative_url }}):
+
+* Added new Covert Ops missions for Tiberian Dawn: "Eviction Notice" and "Twist of Fate"
+* Added new Red Alert missions "Allies10b" and "Soviet13a" and the Counter-Strike mission "Mousetrap"
+* Fixed several palette remapping issues and restored player color brightness remapping
```
Comment (pchote) [inline news/_posts/2023-07-31-playtest-20230731.md:14]: This is underselling the key point here. I suggest rewording to simply "Restored ability to select darker player colours".

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+* Implemented a new custom Lua wrapper
+* Introduced a new modular asset installer
```
Comment (pchote) [inline news/_posts/2023-07-31-playtest-20230731.md:16]: I believe this rework also enabled installing the classic assets from the remaster. IMO that is a more relevant detail to mention here rather than a modder-specific feature.

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+* Reworked sprite sequences, enabling better integration of custom file formats
```
Comment (pchote) [inline news/_posts/2023-07-31-playtest-20230731.md:17]: This line could then become something like "Significantly improved modding support for sequences and asset installation".

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+One of the smaller features is a new smoke emmiter for D2K's damaged vehicles which better matches the original.
```
Comment (pchote) [inline news/_posts/2023-07-31-playtest-20230731.md:19]: typo: emitter.

Comment (pchote) [reply news/_posts/2023-07-31-playtest-20230731.md:19]: And this would usually be put as a figure caption?

File: news/_posts/2023-07-31-playtest-20230731.md (hunk 6 above)
Comment (PunkPun) [reply news/_posts/2023-07-31-playtest-20230731.md:16]: I don't think we should include this in the announcement without being 1005 sure

Comment (pchote) [reply news/_posts/2023-07-31-playtest-20230731.md:16]: https://github.com/OpenRA/OpenRA/blob/bleed/mods/ra/installer/steam.yaml

File: gamenews [hunk 9]
```diff
@@ -1,9 +1,4 @@
...
-	Content: The first tournament on the new release is upon us: on March 18 at 17:00 UTC we will have our 6th edition of the Kautious Kup for Red Alert. If you would like to sign up or find more information, head over to the official Discord server and check out \#competitions-events.
-	
-2023-02-25 18-00-00:
-	Title: Release 20230225
+2023-07-31 22-00-00:
+	Title: Playtest 20230731
 	Author: PunkPun
-	Content: Today we have a special occasion on our hands. We have been waiting for this moment for almost two years: A new stable OpenRA Release is out now, containing several major new features and many small-scale polish improvements. Head to www.openra.net for the full news post and to download the new release.
+	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • Red Alert balance changes\n • And loads of small fixes and tiny features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on forum, Discord, or GitHub. Happy Playtesting!
```
Comment (abcdefg30) [inline gamenews:4]:
```suggestion
	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • Red Alert balance changes\n • And loads of small fixes and tiny features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on the forum, Discord, or GitHub. Happy playtesting!
```

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+As summer is in full swing, we are excited to kick off the playtest series for the upcoming OpenRA release! This time around, it is all about spring cleaning: squashing bugs, implementing minor requested features, and preparing for larger reworks. But fear not! We still have a bunch of cool stuff lined up just for you! Without further ado, here are some notable changes you can expect in [Playtest 20230731]({{ '/download' | relative_url }}):
+
+* Added new Covert Ops missions for Tiberian Dawn: "Eviction Notice" and "Twist of Fate"
+* Added new Red Alert missions "Allies10b" and "Soviet13a" and the Counter-Strike mission "Mousetrap"
+* Fixed several palette remapping issues and restored ability to select darker player colors
+* Implemented a new custom Lua wrapper, improving error reporting for map scripters
+* Introduced a new modular asset installer
+* Reworked sprite sequences significantly improving modding support and asset installation
```
Comment (abcdefg30) [inline news/_posts/2023-07-31-playtest-20230731.md:17]:
```suggestion
* Reworked sprite sequences to significantly improve modding support and asset installation
```

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+<figure>
+  <video autoplay loop muted>
+    <source src="{{ '/images/news/20230731-smoke.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20230731-smoke.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20230731-smoke.png' | relative_url }}"/>
+  </video>
+  <figcaption>One of the smaller features is a new smoke emitter for D2K's damaged vehicles which better matches the original</figcaption>
```
Comment (abcdefg30) [inline news/_posts/2023-07-31-playtest-20230731.md:25]:
```suggestion
  <figcaption>One of the smaller features is a new smoke emitter for D2k's damaged vehicles which better matches the original</figcaption>
```

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+</figure>
+
+For more information see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/).
```
Comment (abcdefg30) [inline news/_posts/2023-07-31-playtest-20230731.md:28]:
```suggestion
For more information see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/51c3790a91a452a4ea42eb95e08618e055f740ee).
```

File: news/_posts/2023-07-31-playtest-20230731.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+And before we wrap up, we would like mention that this release contains significant progress towards C&C Remastered Collection support, including the installation of art assets from the new EA App. "Tiberian Dawn HD" playtest is a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20230731). This preview is multiplayer-compatible with the main 20230731 playtest. However, please keep in mind that performance, memory usage and loading times have not been optimized yet. The C&C Remastered Collection must be installed through Steam or the EA App, and incase you are using macOS or Linux the project README provides you [detailed installation instructions.](https://github.com/OpenRA/TiberianDawnHD#asset-installation)
```
Comment (abcdefg30) [inline news/_posts/2023-07-31-playtest-20230731.md:30]:
```suggestion
And before we wrap up, we would like mention that this release contains significant progress towards C&C Remastered Collection support, including the installation of art assets from the new EA App. The "Tiberian Dawn HD" playtest is a separate release and can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/release-20230731). This preview is multiplayer-compatible with the main 20230731 playtest. However, please keep in mind that performance, memory usage and loading times have not been optimized yet. The C&C Remastered Collection must be installed through Steam or the EA App, and in case you are using macOS or Linux the project README provides you [detailed installation instructions](https://github.com/OpenRA/TiberianDawnHD#asset-installation).
```

**Review by abcdefg30 — APPROVED** [review]
